e2e/configmap.go

```diff
@@ -76,7 +76,11 @@ func createConfigMap(pluginPath string, c kubernetes.Interface, f *framework.Fra
 }
 
 // createCustomConfigMap provides multiple clusters information.
-func createCustomConfigMap(c kubernetes.Interface, pluginPath string, subvolgrpInfo map[string]string) error {
+func createCustomConfigMap(
+	c kubernetes.Interface,
+	pluginPath string,
+	clusterInfoMap map[string]string,
+	skipSubvolFill bool) error {
```

Comment:
This change is not flexible either clusterInfoMap should be a map[string]map[string]string{}  with key as clusterID and the other map contains the subvolumeGroup and radosnamespace. 
 sample code

```suggestion
clusterInfo:=map[string]map[string]string{}
clusterInfo[clusterID1] = map[string]string{}
clusterInfo[clusterID1] ["subvolumeGrop"]="subvolumegroup1"
clusterInfo[clusterID1] ["radosnamepsace"]="radosns1"

clusterInfo[clusterID2] = map[string]string{}
clusterInfo[clusterID2] ["subvolumeGrop"]="subvolumegroup2"
clusterInfo[clusterID2] ["radosnamepsace"]="radosns2"
```

Based on the clusterID fill clusterID details if the map contains the subvolume update subvolumegroup for the clusterID and same can be done for radosnamespace also